ftw/contentpage/tests/pages.py

```diff
@@ -1,45 +1,48 @@
-from ftw.testing import browser
-from ftw.testing.pages import Plone
+from plone import api
 
 INDEX_XPATH = '//*[contains(concat(" ", normalize-space(@class), " "), ' + \
     '" letter-index ")]'
 
 
-class SubjectListingView(Plone):
+class SubjectListingView(object):
+    
+    def __init__(self, browser):
```

Comment:
No, in regular tests this is not required, since the tests (`@browsing`) uses the global default browser singleton instance, which is the same then.

This means that there is probably no test (yet), which actually passes in another instance than default browser instance.

However, I'd still keep making it possible to pass the browser in becuase 1) it is a good habit and 2) other code may import and reuse the page objects.